ui/components/multichain/import-nfts-modal/import-nfts-modal.test.js

```diff
@@ -49,6 +49,10 @@ describe('ImportNftsModal', () => {
     jest.restoreAllMocks();
   });
 
+  // afterEach(() => {
+  //   jest.clearAllMocks();
+  // });
```

Comment:
To remove

Reply:
https://github.com/MetaMask/metamask-extension/pull/32020/commits/f020c9b0ec425014204b33361bbfa917e36d1422